packages/video_player/video_player/lib/video_player.dart

```diff
@@ -661,7 +668,10 @@ class _VideoPlayerState extends State<VideoPlayer> {
   Widget build(BuildContext context) {
     return _textureId == VideoPlayerController.kUninitializedTextureId
         ? Container()
-        : _videoPlayerPlatform.buildView(_textureId);
+        : Transform.rotate(
```

Comment:
We should probably make a helper that only wraps with a `Transform` if there's a non-zero rotation.

Reply:
Agreed. Done.